__tests__/browser.js

```diff
@@ -51,5 +64,89 @@ describe('browser', () => {
       })
       expect(errMessage).toBe('Not Found')
     })
+
+    it('trying to upload without specifying content-type fails', async () => {
+      const errMessage = await evalClient(async (client, uploadContent) => {
+        try {
+          await client.bzz.upload(uploadContent, {})
+        } catch (err) {
+          return err.message
+        }
+      }, uploadContent)
+      expect(errMessage).toBe('Bad Request')
+    })
+
+    it('uploads/downloads the file using bzz', async () => {
+      let evalResponse
+      const manifestHash = await evalClient(async (client, uploadContent) => {
+        const headers = { 'Content-Type': 'text/plain' }
+        return await client.bzz.upload(uploadContent, headers)
+      }, uploadContent)
+      evalResponse = await evalClient(async (client, manifestHash) => {
+        const response = await client.bzz.download(manifestHash)
+        return response.text()
+      }, manifestHash)
+      expect(await evalResponse).toBe(uploadContent)
```

Comment:
I think you can simply do `expect(evalResponse).toBe(uploadContent)` as the `evalResponse` assignment above already uses `await`.